calcom/cal.com
# feat: Add Formbricks experience management support

## What does this PR do?



As discussed with Peer: This PR integrates the [Formbricks](https://formbricks.com) SDK (`@formbricks/js`) into the cal.com platforms and enables running targeted in-app surveys to help understand the needs of the cal.com users.

The SDK loads non-blocking on the client-side and is only enabled when the `NEXT_PUBLIC_FORMBRICKS` environment variable is set. The SDK is only loaded within the authenticated section of the app (after login) and sends user attributes and specific actions the user performs to Formbricks to enable targeted user surveys.

<img width="1493" alt="Screenshot 2023-08-09 at 11 24 32" src="https://github.com/calcom/cal.com/assets/675065/2308b791-0f97-438f-a85d-df5ab6accc56">


Fixes # (issue)



## Type of change




## How should this be tested?



- Create a new product on [app.formbricks.com](https://app.formbricks.com) (or the self-hosted version if this applies)
- Set the `NEXT_PUBLIC_FORMBRICKS` environment variable. It should have the format: `[apiHost]+[environmentId]`, e.g. `NEXT_PUBLIC_FORMBRICKS=https://app.formbricks.com+cll24iwen02vco30hy231njcb`
- Log into cal.com and see a new person getting registered in Formbricks
- Trigger a new action by updating event types or availability in cal.com
- Create a new in-product survey in Formbricks that is getting triggered by one of these new actions
- See the survey showing in cal.com when user performs one of the actions (survey gets only shown once by default)

## Mandatory Tasks
